test/parser/Makefile

```diff
@@ -2,6 +2,7 @@ ifeq ($(origin TOP), undefined)
 	TOP = $(shell git rev-parse --show-toplevel)
 endif
 
+DEF_KORE =
```

Comment:
Why did you add this variable?

Reply:
Oh, actually it looks like it's only needed in `regression-evm` and `regression-wasm`